### PR TITLE
chore: minor typos in the docs

### PR DESCRIPTION
**Pull request by stevesweets**
Just spotted them. 

Great project, has been super useful to me, thanks.

### CODE REVIEW

**Comment by lykmapipo** [comment]
@stevesweets Appreciated.

File: README.md [hunk 1]
```diff
@@ -84,8 +84,8 @@ Example:
 
 ....
 
-const user = user.fakeOnly('name');
-const user = user.fakeOnly('name', 'age');
+const user = User.fakeOnly('name');
```
Comment (lykmapipo) [inline README.md]: @stevesweets This are instance level methods to let update `user` instance with changes.

File: README.md [hunk 2]
```diff
@@ -96,8 +96,8 @@ Example:
 
 ...
 
-const user = user.fakeExpect('name', 'age');
-const users = user.fakeExpect('name', 'age');
+const user = User.fakeExcept('name', 'age');
```
Comment (lykmapipo) [inline README.md]: @stevesweets This are instance level methods to let update `user` instance with changes.

Comment (stevesweets) [reply README.md]: Ah, my mistake. Will review and update

**Pull request closed without merging**